lti_launch_monitor/signals.py

```diff
@@ -0,0 +1,34 @@
+"""
```

Comment:
I think it would be more appropriate to rename this as either `receivers` or `signal_handlers`